### ISSUE TITLE
FR: Record word context by default in Vocabulary Builder 

### ISSUE THREAD

**Issue by 1over137** [koreader/koreader#10064]
**Does your feature request involve difficulty completing a task? Please describe.**
I'm building [VocabSieve](https://github.com/FreeLanguageTools/vocabsieve), a vocabulary learning tool which allows you to create Anki cards from a variety of sources. A major feature is importing words looked up and/or highlighted from KOReader and Kindle. Currently, the KOReader importing function is achieved by parsing the metadata.lua files, and going through each book and hunting down the sentence through xpath. While this works for the most part, it is difficult to maintain with respect to a variety of possibly non-compliant ebook formats. It would be great if this feature is on by default (which is what Kindle does). It doesn't consume very much storage (comparable to one extra book with a cover image), and having it on by default would be a better experience for users, when there is no simple way to retrieve the sentences again for lookups where the context was not recorded.

**Comment by weijiuqiao** [comment]
It has been on secretly by default for sometime. Though for users that had saved it to be otherwise (either by choice or when it was off by default), it's still off:
https://github.com/koreader/koreader/blob/747c3eaf9d72414d272f5de7c743e4a0153c9012/plugins/vocabbuilder.koplugin/main.lua#L60

Also, the Vocab Builder plugin saves words and context to a sqlite db. If you rely on it, there's maybe no need to go through the books.

**Comment by 1over137** [comment]
Yeah, the point is to not have to go through the books. Wait, does that mean Vocab builder itself is off by default?

**Comment by weijiuqiao** [comment]
No. The `enabled` here means automatically adding new words to Vocab builder is off by default. Users need to tap the "add to vocabulary builder" manually when looked up a word.

**Comment by 1over137** [comment]
Though, I'm using the database for two things simultaneously. 
1. Logging all the lookups to the Vocabsieve database combined with lookups from all other sources, to estimate the user's familiarity with the word, which is useful for purposes like analyzing book difficulty.
2. Provide the ability to import them as notes, possibly filtered by whether or not they are highlighted in the book.

**Comment by weijiuqiao** [comment]
>Or is the lookup history separate from Vocab builder?

Yes.
